### DISCUSSION TITLE
RedHat and iPXE boot

### DISCUSSION THREAD

**Discussion by lucziv** [ipxe/ipxe#830] [General]
Hello Folks,

I am trying to boot my RedHat. I have enabled PXE boot on given port, however I am constantly getting below error message.

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/119680612/205389435-f5afafd1-7eb7-4e1e-bccd-1e3215270baf.png)
![MicrosoftTeams-image](https://user-images.githubusercontent.com/119680612/205389455-8b2a373a-448f-49e5-be64-72b0a43cd91e.png)

I faced PCI error and I really don't understand how it can be related to iPXE. Could you please help me on this? Thanks!

**Comment by NiKiZe** [comment]
As you say, how is this related to iPXE?
Have you verified that you don't get this when booting from usb?

**Reply by lucziv** [reply]
No errors from USB, but I have these errors in Legacy and UEFI mode

**Reply by stappersg** [reply]
Acknowledge on the
> No errors from USB, but I have these errors in Legacy and UEFI mode

And back to the
> How is this related to iPXE?
> Have you verified that you don't get this when booting from USB?

Please tell us more about the verification, like which kernel versions are being used.

**Comment by habetsm-xilinx** [comment]
That is an Intel root port. Very new RedHat releases, so contact them for advice.

**Reply by stappersg** [reply]
Not knowning what _Intel root port_ is, did I a websearch on `8086:7ab9`.
( the `8086:7ab9` is from the repeating line `device [8086:7ab9] error status/mask/00000001/00002000` of the above screenshot )
Among others it resulted in https://pci-ids.ucw.cz/read/PC/8086/7ab9
That web page says it is a _PCI Express Root Port_.
